src/themis/secure_message.h

```diff
@@ -154,8 +154,7 @@ themis_status_t themis_secure_message_verify(const uint8_t* public_key,
  * storage then THEMIS_BUFFER_TOO_SMALL will return and wrapped_message_length will store length of
  * buffer needed for wrapped message store
  */
-DEPRECATED("use 'themis_secure_message_encrypt' with private and public keys to encrypt message, "
-           "or 'themis_secure_message_sign' with private key to sign message")
+DEPRECATED("use 'themis_secure_message_encrypt' with private and public keys to encrypt message, or 'themis_secure_message_sign' with private key to sign message")
```

Comment:
doxygen doesn't support multiline macro/function calls?

Reply:
Well, _not really_. It tries really hard to emulate C preprocessor, but sometimes this leads to poor results. Doxygen does preprocessing mostly to exclude parts of the file due to conditional compilation.

Here the THEMIS_API and DEPRECATED macros are considered a part of return type and they get mentioned in function signature, and this breaks Doxygen parser leading to funny output:

<img width="898" alt="Screen Shot 2019-12-03 at 16 46 00" src="https://user-images.githubusercontent.com/1256587/70061154-75244d80-15ec-11ea-9fdb-531b7bc233b8.png">

Strictly speaking, they _are_ a part of the signature, but we’re not really interested in this when generating API documentation. Thus we’d like to remove both of them.

Enabling EXPAND_ONLY_PREDEF and adding THEMIS_API to PREDEFINED gets rid of it. DEPRECATED macros will be ignored if they are written in one line, thanks to SKIP_FUNCTION_MACROS:

https://github.com/cossacklabs/themis/blob/c8a469fcd0d6e23d0b5f20ea235455c268228c33/Doxyfile#L1987-L1995